pipelines/tekton/aiedge-e2e/tasks/test-model-rest-svc-task.yaml

```
@@ -31,8 +31,7 @@ spec:
         # Check response:
         echo "Check response:"
         cat /tmp/output.json
-        # https://stackoverflow.com/a/7167115/19020549
-        grep -Pazo '\"predictions\":\s*\[\s*[-+]?([0-9]*\.[0-9]+|[0-9]+)*\s*\]' /tmp/output.json
+        test "$( python3 -m json.tool --no-indent < $(workspaces.test-data.path)/output.json )" = "$( python3 -m json.tool --no-indent < /tmp/output.json )"
```

This also works
```suggestion
        test "$( python3 -m json.tool --no-indent $(workspaces.test-data.path)/output.json )" = "$( python3 -m json.tool --no-indent /tmp/output.json )"
```
and it allows for a nicer log, explicitly showing from which files the content was read. Currently the log looks like this (from the PR check):
```bash
++ python3 -m json.tool --no-indent
++ python3 -m json.tool --no-indent
+ test '{"predictions": [-28.7639923]}' = '{"predictions": [-28.7639923]}'
```

```bash
++ python3 -m json.tool --no-indent ${some-workspace-path}/output.json
++ python3 -m json.tool --no-indent /tmp/output.json
+ test '{"predictions": [-28.7639923]}' = '{"predictions": [-28.7639923]}'
```
I guess the second example is easier for possible debugging, what do you think?

---

Good point, updated with the input files as parameters.